src/tests/TokenTests.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "../TokenFuzzTests.sol";
+import "./SampleToken.sol";
+
+contract TokenTestsTests is TokenTests, TokenFuzzTests 
+ {
+    function setUp() public {
+        _token_ = address(new SampleToken());
+        _tokenName_ = "Sample Token";
+        _contractName_ = "SampleToken";
+        _symbol_ = "TKN";
+        TokenLike(_token_).deny(address(this));
```

Comment:
There is not mention of this line in the README.

Reply:
The user of the library is free not to deny their test contract. In this case I do it to make sure the tests do not depend on the test contract having been denied. Will add a comment about it.